cmd/kubelet/app/server.go

```diff
@@ -166,7 +167,7 @@ func NewKubeletServer() *KubeletServer {
 		HealthzPort:                 10248,
 		HealthzBindAddress:          util.IP(net.ParseIP("127.0.0.1")),
 		RegisterNode:                true, // will be ignored if no apiserver is configured
-		OOMScoreAdj:                 -900,
+		OOMScoreAdj:                 -1000,
```

Comment:
My only worry here is that the Kubelet could be killed if we launch too many guaranteed pods. The Kubelet and the guaranteed pods would have an OOM score of 0. Do you think that's OK?